pytrip/utils/spc2pdf.py

```diff
@@ -174,42 +176,44 @@ def main(args=sys.argv[1:]):
-                    if z_data.tot_part_number.any():
-                        mask1 = (z_data.depth == depth_step)
-                        energy_bin_centers = z_data.energy_left_edge[mask1] + 0.5 * z_data.energy_bin_width[mask1]
-                        tot_part_number = z_data.tot_part_number[mask1]
+                for az in az_uniq:
+                    a, z = az
+                    az_data = data[(data.z == z) & (data.a == a)]
+                    if az_data.tot_part_number.any():
+                        mask1 = (az_data.depth == depth_step)
+                        energy_bin_centers = az_data.energy_left_edge[mask1] + 0.5 * az_data.energy_bin_width[mask1]
+                        tot_part_number = az_data.tot_part_number[mask1]
                         if np.any(mask1):
                             ax.plot(energy_bin_centers, tot_part_number,
                                     linestyle=linestyle,
                                     marker=marker,
-                                    label="Z = {:d}".format(z))
+                                    label="Z = {:d} A = {:d}".format(z, a))
                 plt.legend(loc=0)
                 pdf.savefig(fig)
                 plt.close()
                 logging.debug("Spectrum for depth {} saved".format(depth_step))
 
         # then couple of pages, each with heatmap plot of spectrum for given particle specie
-        for z in z_uniq:
-            z_data = data[data.z == z]
-            if z_data.tot_part_number.any():
-                depth_steps = z_data.depth[z_data.energy_left_edge == z_data.energy_left_edge[0]]
-                energy_left_edges = z_data.energy_left_edge[z_data.depth == z_data.depth[0]]
-                energy_bin_widths = z_data.energy_bin_width[z_data.depth == z_data.depth[0]]
-
-                zlist = z_data.tot_part_number.reshape(depth_steps.size, energy_left_edges.size).T
+        for az in az_uniq:
+            a, z = az
+            az_data = data[(data.z == z) & (data.a == a)]
+            if az_data.tot_part_number.any():
+                depth_steps = az_data.depth[az_data.energy_left_edge == az_data.energy_left_edge[0]]
+                energy_left_edges = az_data.energy_left_edge[az_data.depth == az_data.depth[0]]
+                energy_bin_widths = az_data.energy_bin_width[az_data.depth == az_data.depth[0]]
+
+                zlist = az_data.tot_part_number.reshape(depth_steps.size, energy_left_edges.size).T
                 if parsed_args.logscale:
                     norm = colors.LogNorm(vmin=zlist[zlist > 0.0].min(), vmax=zlist.max())
                 else:
                     norm = colors.Normalize(vmin=0.0, vmax=zlist.max())
                 fig, ax = plt.subplots()
-                ax.set_title("Spectrum for Z = {}".format(z))
+                ax.set_title("Spectrum for Z = {} A = {}".format(z, a))
                 ax.set_xlabel("Depth [cm]")
-                ax.set_ylabel("Energy [MeV]")
+                ax.set_ylabel("Energy [MeV/amu]")
```

Comment:
sure not MeV/nucleon?

Reply:
in TRiP98 documentation `MeV/u` is usually used: http://bio.gsi.de/DOCS/TRiP98/PRO/DOCS/trip98fmtspcascii.html